src/Domain/CoordinateMaps/TimeDependent/Skew.hpp

```diff
@@ -59,76 +61,22 @@ namespace domain::CoordinateMaps::TimeDependent {
  *
  * When the $\cos$ term is $-1$, then $W = 0$, and when the $\cos$ term is 1,
  * then $W = 1$. Therefore, the function $\lambda(\vec{x})$ must go from 0 at
- * the \p center, to 1 at $R$. In order to derive
- * $\lambda(\vec{x})$, we introduce three vectors, shown in the figure, all
- * originating from the origin: $\vec{x}$ is our source coordinate, $\vec{x}_0$
- * is the \p center, and $\vec{x}_P$ is the vector to the intersection point
- * between the \p outer_radius and the ray connecting $\vec{x}_0$ and $\vec{x}$.
- * Therefore, $|\vec{x}_P| = R$.
- *
- * \image html Skew.svg "Vectors for Skew map"
- *
- * We choose $\lambda(\vec{x})$ to linearly go from 0 at $\vec{x}_0$ to 1 at
- * $\vec{x}_P$ with the form
+ * the origin, to 1 at $R$. We choose $\lambda(\vec{x})$ to quadratically go
+ * from 0 at the origin to 1 at $R$ with the form
  *
  * \begin{equation}
  *   \label{eq:lambda}
- *   \lambda(\vec{x}) = \frac{|\vec{x} - \vec{x}_0|}{|\vec{x}_P - \vec{x}_0|}
+ *   \lambda(\vec{x}) = \frac{|\vec{x}|^2}{R^2}.
```

Comment:
My understanding is that you added the square to make this smooth - if that is the case, I don't see the reasoning there. The square only changes the degree of differentiability by 1, doesn't it? The issue remains that $\frac{dr}{dx^i} = \frac{x^i}{r}$ is not smooth at the origin (e.g. along the x-axis it is -1 when approached from the left and +1 when approached from the right). I think this is a problem, unless we can guarantee that the origin is always on an element boundary.

Reply:
$W$ ***IS*** smooth though so it's fine. Also I've checked first 3 derivatives of $W$ are smooth as well (it's just sine terms multiplied by powers of $x$). Whether $\lambda$ is smooth is irrelevant. Only the derivatives of $W$ are the ones that mater.

Reply:
Ok I'm convinced now that the map is smooth. The square is actually not relevant because the cos is symmetric, so $\lambda(\vec{x}) = \frac{|\vec{x}|}{R}$ would also work.

Reply:
I don't think `|x|/R` works, despite the cosine being symmetric.  Because when you differentiate, you get sine terms, and even worse, you get derivs of `|x|/R`).  `|x|^2/R^2` is fine though.